src/android/aidl/types.md

```diff
@@ -0,0 +1,8 @@
+# Working With AIDL Types
+
+AIDL types translate into the appropriate idiomatic Rust type:
+
+* Primitive types map (mostly) to idiomatic Rust types.
+* Collection types like slices, `Vec`s and string types are supported.
+* References to AIDL objects can be sent between file handles.
```

Comment:
The phrasing here is a little clunky. Do you mean you can send objects, or file handles?

Reply:
I think my brain farted while typing that sentence. Fixed!